automarker_2/automarker_app/management/commands/check_all_configuration.py

```diff
@@ -0,0 +1,96 @@
+import concurrent.futures
+
+from django.core.management.base import BaseCommand, CommandParser
+
+from automarker_app.lib.marker import (
+    get_all_marker_configs,
+    mark_project,
+    get_steps_final_status,
+)
+from automarker_app.lib.constants import (
+    CONFIG_STATUS_DEBUG,
+    CONFIG_STATUS_PRODUCTION,
+    STEP_STATUS_PASS,
+    STEP_STATUS_WAITING,
+)
+
+
+from ..utils import print_steps_result
+
+
+DASH_SEPARATOR = "-" * 100
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument("--skip", type=int, nargs="+")
+
+    def handle(self, *args, **options):
+        good_configs = []
+        bad_configs = []
+
+        skip_ids = options.get("skip", [])
```

Comment:
You can cleanup the cmd, had added some arg to skip certain IDs for faster testing. Not sure if it will still be needed?